src/main/java/uk/gov/hmcts/reform/ccd/document/am/configuration/ApplicationConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.ccd.document.am.configuration;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Slf4j
+public class ApplicationConfiguration {
+
+    private final String s2sSecret;
+    private final String s2sMicroService;
+    private final String s2sUrl;
+    private final String pbaFromUserAccountMap;
+
+    public ApplicationConfiguration(@Value("${idam.s2s-auth.totp_secret}") String s2sSecret,
+                                    @Value("${idam.s2s-auth.microservice}") String s2sMicroService,
+                                    @Value("${idam.s2s-auth.url}") String s2sUrl,
+                                    @Value("${idam.s2s-auth.pbaFromUserAccountMap}") String pbaFromUserAccountMap) {
+        this.s2sSecret = s2sSecret;
+        this.s2sMicroService = s2sMicroService;
+        this.s2sUrl = s2sUrl;
+        this.pbaFromUserAccountMap = pbaFromUserAccountMap;
+        //log.info("Configured S2S secret: " + s2sSecret.substring(0, 2) + "************" + s2sSecret.substring(14));
```

Comment:
As per previous security incidents we are not supposed to write the s2s key and URLs in any logging, thats the reason it commented, could you please remove this commented logging.